com.playonlinux.PlayOnLinux4.yaml

```diff
@@ -0,0 +1,148 @@
+    sources:
+      - type: archive
+        url: https://imagemagick.org/archive/releases/ImageMagick-7.1.1-14.tar.gz
+        sha256: 2f52b6c9e4e0dfd8789e409b110dba2e806956ca96fb1b3c367887ff40ffb2d8
+        x-checker-data:
+            type: anitya
+            project-id: 1372
+            url-template: https://imagemagick.org/archive/releases/ImageMagick-$version.tar.gz
+
+  - name: icoutils
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/icoutils/icoutils-0.32.3.tar.bz2
+        sha256: 17abe02d043a253b68b47e3af69c9fc755b895db68fdc8811786125df564c6e0
+        x-checker-data:
+            type: anitya
+            project-id: 1360
+            url-template: http://download.savannah.gnu.org/releases/icoutils/icoutils-$version.tar.bz2
+
+  - name: oniguruma
+    buildsystem: autotools
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://api.github.com/repos/kkos/oniguruma/tarball/v6.9.8
+        sha256: 1890b8c3150b65a6d761a72e3212a98d6b9c119a469c1c93099234bd3d6c5a0e
+        x-checker-data:
+            type: json
+            url: https://api.github.com/repos/kkos/oniguruma/releases/latest
+            version-query: .tag_name
+            url-query: .tarball_url
+
+  - name: jq
+    buildsystem: autotools
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://api.github.com/repos/jqlang/jq/tarball/jq-1.6
+        sha256: 30eaa22720cd899fd6b9b791bb079b0ae2745f9e3db59fd6730c807300c98156
+        x-checker-data:
+            type: json
+            url: https://api.github.com/repos/jqlang/jq/releases/latest
+            version-query: .tag_name
+            url-query: .tarball_url
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -fi
+
+  - xterm.yaml
+
+  - scrot.yaml
+
+  - name: fake-sudo
+    buildsystem: simple
+    build-commands:
+      - echo '#!/bin/sh' > $FLATPAK_DEST/bin/sudo
+      - chmod +x $FLATPAK_DEST/bin/sudo
+
+  - name: PlayOnLinux
+    buildsystem: simple
+    build-commands:
+      - rm -rf $FLATPAK_DEST/share/locale
+      - mkdir -p $FLATPAK_DEST/bin
+      - ln -s /usr/bin/python $FLATPAK_DEST/bin/python2
+      - PREFIX=$FLATPAK_DEST make build
+      - PREFIX=$FLATPAK_DEST make install
+      - install -Dm644 $FLATPAK_DEST/share/pixmaps/playonlinux.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 $FLATPAK_DEST/share/pixmaps/playonlinux32.png $FLATPAK_DEST/share/icons/hicolor/32x32/apps/$FLATPAK_ID.png
+      - install -Dm644 $FLATPAK_DEST/share/pixmaps/playonlinux16.png $FLATPAK_DEST/share/icons/hicolor/16x16/apps/$FLATPAK_ID.png
+      - mv $FLATPAK_DEST/share/applications/PlayOnLinux.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - desktop-file-edit --set-name "PlayOnLinux 4" --set-icon $FLATPAK_ID $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
```

Comment:
you should `rename-icon` and `rename-desktop-file`

it will take care of this.

Reply:
Done for `rename-desktop-file`. I need to call `desktop-file-edit` anyway, so I can also use it to change the Icon.

Reply:
I know, but the Icon is already renamed, so I don't need this.